app/src/renderer/components/common/AnchorCopy.vue

```diff
@@ -31,6 +30,10 @@ export default {
       required: true
     }
   },
+  data: () => ({
+    /* istanbul ignore next */
```

Comment:
curious about what's `istanbul`

Reply:
coverage reporter, this ignores the following line